src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -50,6 +50,10 @@
                     <loading-spinner size="xs"></loading-spinner>
                     <span v-translate="'loadingDatasets'"></span>
                 </div>
+                <div v-if="adrError" id="fetch-error">
+                    <div v-translate="'errorFetchingDatasetsFromADR'"></div>
```

Comment:
This could have been our standard error view I guess, but perhaps better to have a generic error message as this is an expected intermittent error.